src/workspace/workspace.service.ts

```diff
@@ -16,4 +17,16 @@ export class WorkspaceService {
 			throw new CustomError('Failed to get workspace counts', err, { url });
 		}
 	}
+
+	public static getAllBookmarksForUser(): Promise<BookmarkInfo[]> {
+		let url: string | null = null;
+		try {
+			url = (getEnv('PROXY_URL') as string) + '/workspace/bookmarksForUser';
```

Comment:
Most paths in the avo proxy use kebabcase for their routes
Also the "ForUser" seems quite obvious to me, since we're inside the workspace route
So i would switch it to:
```suggestion
			url = (getEnv('PROXY_URL') as string) + '/workspace/bookmarks';
```